snappy_wrappers/wrappers/pvactools/combining/comb_rna.py

```diff
@@ -0,0 +1,530 @@
+                args.input_vcf
+            )
+        )
+    # snv part
+
+    return vcf_reader, rna_vcf_reader, is_multi_sample
+
+
+########################## Adding extra fields into Format ##########################
+def create_vcf_writer(args, vcf_reader):
+    (head, sep, tail) = args.input_vcf.rpartition(".vcf")
+    new_header = vcf_reader.header.copy()
+    if args.mode == "gene":
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "GX"),
+                    ("Number", "."),
+                    ("Type", "Float"),
+                    ("Description", "Gene Expressions"),
+                ]
+            )
+        )
+        output_file = ("").join([head, ".gx.vcf", tail])
+    elif args.mode == "transcript":
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "TX"),
+                    ("Number", "."),
+                    ("Type", "Float"),
+                    ("Description", "Transcript Expressions"),
+                ]
+            )
+        )
+        output_file = ("").join([head, ".tx.vcf", tail])
+    if args.rna_vcf:
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "RAD"),
+                    ("Number", "."),
+                    ("Type", "Integer"),
+                    (
+                        "Description",
+                        "Allelic depths from mRNA sequencing data of the same sample",
+                    ),
+                ]
+            )
+        )
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "ROT"),
+                    ("Number", "."),
+                    ("Type", "Integer"),
+                    (
+                        "Description",
+                        "Sum of allelic depths of other alternative allele from mRNA sequencing data of the same sample",
+                    ),
+                ]
+            )
+        )
+        (head, sep, tail) = output_file.rpartition(".vcf")
+        output_file = ("").join([head, ".pileup.vcf", tail])
+    if args.output_vcf:
+        output_file = args.output_vcf
+    return vcfpy.Writer.from_path(output_file, new_header)
+
+
+########################## Auto assign gene id column based on different tools ##########################
+def resolve_id_column(args):
+    if args.format == "cufflinks":
+        return "tracking_id"
+    elif args.format == "kallisto":
+        if args.mode == "gene":
+            return "gene"
+        elif args.mode == "transcript":
+            return "target_id"
+    elif args.format == "stringtie":
+        if args.mode == "gene":
+            return "Gene ID"
+    elif args.format == "star":
+        return "gene_id"  # NEED change
+    elif args.format == "custom":
```

Comment:
Can the user set the number of the id column? Same for expression. See other comments below